ui/web/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "devDependencies": {
+    "jquery": "3.3.1",
+    "jqplot": "1.0.9"
+  }
+}
```

Comment:
Not let's add some instruction about how to use our application as web server, it should contain something like this:

To run this application, please do:
1) Install 'npm`;
2) Navigate to `work_estimation/ui/web` folder and run: `npm install`;
3) Install `python 3`;
4) Install `pip`;
5) Navigate to `work_estimation` folder and load python packages by next command: `pip install requirements.txt`
6) Navigate to `work_estimation` folder and run `work_estimation_console_adapter.py` python script that starts web server;
7) Open `http://localhost:5000/application/index.html`.

This instruction should be placed in `readme.md` file that should be placed in `work_estimation/ui/web` folder. Such files have their specific syntax, please refer to [https://dillinger.io/](https://dillinger.io/) for examples. Such files is rendered automatically on github, here is example: [https://github.com/shnax0210/work_estimation/blob/master/readme.md](https://github.com/shnax0210/work_estimation/blob/master/readme.md)

Reply:
Added readme.md file

Reply:
Great! Let's also highlight folder and file names and console commands by using wrapping it by ` symbol. 

For example \`npm install\` it will be rendered then as `npm install`.

Reply:
Also sorry, it's my mistake but we need to run `work_estimation_web_adapter.py` instead of `work_estimation_console_adapter.py`.

Reply:
Added changes to the readme.md file